src/components/Stories/Stories.tsx

```diff
@@ -48,13 +51,21 @@ export function Stories({
         [onClose],
     );
 
+    const {callback: closeWithLS} = useSyncWithLS<NonNullable<StoriesProps['onClose']>>({
+        callback: (event, reason) => {
+            if (syncInTabs) onClose?.(event, reason);
```

Comment:
Is it necessary to check `syncInTabs` here?

Reply:
Tnx, fixed.